region/brinstar/red/Caterpillar Room.json

```diff
@@ -2214,6 +2230,45 @@
         "Morph"
       ],
       "flashSuitChecked": true
+    },
+    {
+      "link": [7, 5],
+      "name": "Exit G-Mode",
+      "requires": [
+        "canEnterGMode"
+      ],
+      "flashSuitChecked": true,
+      "note": "Exit G-Mode near the door to pass the gate."
+    },
+    {
+      "link": [7, 5],
+      "name": "Exit G-Mode, Open Gate",
+      "requires": [
+        "canEnterGMode",
+        "canOffScreenMovement"
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "note": [
+        "Exit G-Mode while under the gate to open it without a Super.",
+        "The camera will not scroll through the morph tunnel, but Samus will still be visible;",
+        "position where Samus appears to be standing on air one tile to the right of the ledge.",
+        "With Morph, Samus can fix the camera by going through the morph tunnel."
+      ]
+    },
+    {
+      "link": [7, 6],
+      "name": "Carry G-Mode Up Elevator",
```

Comment:
I'm probably missing something, but I don't understand what this strat is for. What would be the purpose of going to this G-mode morph tunnel junction only to then go up the elevator? It looks like there are already strats that go from each of the doors to the elevator (including one from node 5).

In this case, it could make more sense to define node 7 as being at the right door rather than in the morph tunnel, since its only purpose is to either go past the gate or to open the gate? At least with how it's written now, I'm not seeing a purpose to go left from node 7?

Reply:
i was mostly thinking because this is where the off screen movement starts, but I don't think it should really matter. I think [7, 6] should be removed